spec/library_spec.rb

```diff
@@ -0,0 +1,35 @@
+require './lib/library.rb'
+
+describe Library do
+books = [{ title: 'Alfons leker', author: 'A. Andersson' },{ title: 'Bible', author: 'TGAOTU' } ]
+  subject { described_class.new(books: books)}
+  #let(:user) { double('User') }
+
+
+ it 'has a collection of books' do
+    expect(subject.books).to be_kind_of Array
+  end
+
+#it 'sets due date in one month from today' do |variable|
+
+#end
+
+ it 'finds and allows check out by #title' do
+    item = { title: 'Alfons leker'}
+    response = { item: { title: 'Alfons leker', author: 'A. Andersson' },
+                # lender: user,
+                 return_date: Date.today.next_month.strftime('%F')}
+#binding.pry
+    expect(subject.books.lend(item)).to eq response
+  end
```

Comment:
Lint/BlockAlignment: `end` at 24, 2 is not aligned with `it 'finds and allows check out by #title' do` at 17, 1.